integration_tests/pages/page.ts

```diff
@@ -0,0 +1,17 @@
+export type PageElement = Cypress.Chainable<JQuery>
+
+export default abstract class Page {
+  static verifyOnPage<T>(constructor: new () => T): T {
+    return new constructor()
+  }
+
+  constructor(private readonly title: string) {
+    this.checkOnPage()
+  }
+
+  checkOnPage(): void {
+    cy.get('h1').contains(this.title)
```

Comment:
They made an interesting change on create-and-vary-a-licence to use an id to verify the page rather than the title - making the pages slightly more robust (titles change - ids don't). Might be worth switching to that in a future PR?

Reply:
cool thanks. I might look at doing that in my next PR. I've got lots more to do in cypress!